locale/zh-tw/about/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: about.hbs
+title: 關於
+trademark: 商標
+---
+
+# 關於 Node.js®
+
+作為一個異步事件驅動的 JavaScript 執行階段，Node.js 設計來建構可擴充的網路應用程式。下方的「Hello World」範例中，多個連線皆能並行處理。一當每個連線建立後，會先執行回呼函式，但如果沒有需要作的事情，Node.js 則會休眠。
+
+```javascript
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`伺服器正在 http://${hostname}:${port}/ 運作`);
+});
+```
+
+This is in contrast to today's more common concurrency model, in which OS threads
```

Comment:
Is this missing translation?

Reply:
No, I just left them untranslated as I don't have time. :(